src/components/profiles/wallet-hypha.vue

```diff
@@ -0,0 +1,100 @@
+<script>
+import SimpleCrypto from 'simple-crypto-js'
+
+export default {
+  name: 'wallet-hypha',
+  components: {
+    TreasuryToken: () => import('~/components/organization/treasury-token.vue'),
+    Widget: () => import('~/components/common/widget.vue')
+  },
+
+  props: {
+    account: {
+      type: String,
+      default: null
+    },
+
+    balances: {
+      type: Array,
+      default: () => []
+    },
+
+    isAdmin: {
+      type: Boolean,
+      default: false
+    },
+
+    quantity: Number
+  },
+
+  computed: {
+    canActivate () { return true },
+    hasEnoughTokens () { return Number(this.balances?.[0]?.amount) >= this.quantity }
```

Comment:
@arsenijesavic would be nicer to have computed `tokens` that returns `Number(this.balances?.[0]?.amount)` and then `hasEnoughTokens` would be `return this.tokens >= this.quantity`